### ISSUE TITLE
LA100.IMAGE_downscale_ctry_yr

### ISSUE THREAD

**Issue by bpbond** [JGCRI/gcamdata#111]

**Comment by bpbond** [comment]
Hi @crvernon it looks like this file uses a `downscale_IMAGE_regions` routine. Please check whether it's called elsewhere in the data system; if not, it can be defined locally (in your new file), but if yes, please put it in a separate file--we can talk. Thanks.

**Comment by bpbond** [comment]
@crvernon @kvcalvin Should this chunk be reassigned to someone else? Don't know your (CV) time commitments, but it would be good to get it done one way or another.

**Comment by crvernon** [comment]
@bpbond @kvcalvin Apologies.  I have been swamped.  I will let you know by Wednesday, May 10 COB about getting to this.  If I cannot get a chance to do it by then, I will get you to reassign.  I am interested in helping out on this.  Thanks.

**Comment by crvernon** [comment]
@bpbond @kvcalvin Still working to make time for this.  If this this chunk is a dependency for another where someone is waiting, I would reassign.  If not, I will try to get to it by the end of the week.  Thanks.

**Comment by bpbond** [comment]
OK, I'm reassigning this. Chris, if you get time, please talk to Kate about re-engaging with DSR.

**Comment by bpbond** [comment]
@pkyle any comments on the metadata for `IMAGE_an_Feedfrac_Rimg_C_Sys_Fd_Y.csv`, `IMAGE_an_FeedIO_Rimg_C_Sys_Y.csv`, and `IMAGE_an_Prodmixfrac_Rimg_C_Y.csv`? In particular, units? Here's what there is currently:

```
# File: IMAGE_an_Feedfrac_Rimg_C_S_F_Y.csv
# Title: IMAGE animal feed fractions by IMAGE region / animal type / feeding system / feed type / and year
# Units: NA
# Author: Page Kyle
# Original source: aglu-data/RawData/IMAGE/Agr Production.xls [Agr Production]

# File: IMAGE_an_FeedIO_Rimg_C_S_F_Y.csv
# Title: IMAGE animal feed input-output coefficients by IMAGE region / animal type / feeding system / feed type / and year
# Units: NA
# Author: Page Kyle
# Original source: aglu-data/RawData/IMAGE/Input Fractions.xls [IO Coeffs]

# File: IMAGE_an_Prodmixfrac_Rimg_C_Y.csv
# Title: IMAGE animal production system fractions by IMAGE region / animal type / and year
# Units: NA
# Author: Page Kyle
# Source: aglu-data/RawData/IMAGE/Fraction Mixed.xls [Fraction Mixed]
```

Thanks!

**Comment by pkyle** [comment]
For the most part, that information is correct, except for the Author and Original source. The NA units could be replaced with "unitless" on Feedfrac and Prodmixfrac, and with "dry feed to animal commodity ratio" for FeedIO. The data source for all three is the IMAGE model, version 2.4. These tables were once downloaded from the model's website, but are no longer available. Still for the source, we can point to the IMAGE 2.4 documentation; within the document, the relevant part is Figure 5.1 (p80) and the surrounding text.
Bouwman et al., 2006, Integrated Modelling of Global Environmental Change: An Overview of IMAGE 2.4. Netherlands Environmental Assessment Agency, MNP Publication 500110002/2006. https://www.researchgate.net/profile/Alexander_Bouwman/publication/235704246_Integrated_modelling_of_global_environmental_change_an_overview_of_Image_24/links/57a4307808ae455e8535f763.pdf

**Comment by bpbond** [comment]
Thanks @pkyle !

**Comment by bpbond** [comment]
Merged in #443
